docs/docs/how_to/vectorstores.mdx

```diff
@@ -55,6 +55,41 @@ pip install langchain-chroma
 from langchain_chroma import Chroma
 
 db = Chroma.from_documents(documents, OpenAIEmbeddings())
+```
+
+  </TabItem>
+  <TabItem value="pinecone" label="Pinecone">
+
```

Comment:
All the current vectorstores features here are local vectorstores if I'm not mistaken. We did this on purpose to avoid a race between vectorstores providing to populate the tabs on this page. cc @efriis